misk-redis/src/main/kotlin/misk/redis/Redis.kt

```diff
@@ -746,6 +753,30 @@ interface Redis {
 
         return params
       }
+
+      /**
+       * Checks that options will result in a valid ZADD command, conforming to the following:
+       *
+       * ```
+       * zadd key [NX|XX] [GT|LT] [CH] score member [score member ...]
+       * ```
+       *
+       * See the [ZADD command documentation](https://redis.io/commands/zadd) for more information.
+       */
+      internal fun verify(options: Array<out ZAddOptions>) {
```

Comment:
Jedis 5 no longer throws this exception for zadd. Keeping this behavior in misk for backwards compatibility reasons. I don't really know how Jedis 5 resolves incompatible options, but I think checking before making the actual call is a good idea anyway.